3rdparty/ippicv/ippicv.cmake

```diff
@@ -2,7 +2,7 @@ function(download_ippicv root_var)
   set(${root_var} "" PARENT_SCOPE)
 
   # Commit SHA in the opencv_3rdparty repo
-  set(IPPICV_COMMIT "7f55c0c26be418d494615afca15218566775c725")
+  set(IPPICV_COMMIT "d1cbea44d326eb0421fedcdd16de4630fd8c7ed0")
```

Comment:
- [ ] Shouldn't it be different for x86 and x86_64 as only x86_64 ICV packages are available since IPP 2022.0.0?